File: examples/LINUX/benchmarks/programs/function-leap-year.py
def leap_year(y):
  r = 0
  if(y % 4 == 0 and y % 100 != 0) or (y % 400 == 0):
    r = True
  else:
    r = False
  return r

i = 0
for i in range(100000):
    i = leap_year(i)

File: examples/LINUX/benchmarks/programs/function-leap-year.php
<?php

$i = 0;
for($y = 0; $y < 100000; $y++)
  $i = leap_year($y);

function leap_year($y) {
  $r = 0;
  if(($y % 4 == 0) && ($y % 100 != 0) || ($y % 400 == 0)) $r = true;
  else $r = false;
  return $r;
}
